certauth.go

```diff
@@ -77,6 +80,22 @@ func (a *Auth) Handler(h http.Handler) http.Handler {
 	})
 }
 
+// RouterHandler implements the httprouter.Handle for integration with github.com/julienschmidt/httprouter
+func (a *Auth) RouterHandler(h httprouter.Handle) httprouter.Handle {
```

Comment:
@spheromak yes, and then you lose the ability to do params. Also you can't use any of the httprouter.GET(/route/:param) funcs, as far as I can tell.